charts/grafana/values.yaml

```diff
@@ -504,6 +504,71 @@ datasources: {}
 #        authType: default
 #        defaultRegion: us-east-1
 
+## Configure grafana alerting (can be templated)
+## ref: http://docs.grafana.org/administration/provisioning/#alerting
+##
+alerting: {}
+#  rules.yaml: |
+#    apiVersion: 1
+#    groups:
+#      - orgId: 1
+#        name: my_rule_group
```

Comment:
We should demo the user the yaml we passed in could be self-referable, otherwise the user would question why they need to raw string the ``{{  `{{ ... }}` }}``:

```suggestion
#        name: {{ .Chart.Name }}_my_rule_group
```

Reply:
ok, can add this in the examples.

Reply:
adapted